plugins/googlesignin/class.googlesignin.plugin.php

```diff
@@ -22,6 +22,7 @@ class GoogleSignInPlugin extends Gdn_OAuth2 {
     public function __construct() {
         $this->setProviderKey('googlesignin');
         $this->settingsView = 'plugins/settings/googlesignin';
+        $this->authenticationScheme = 'googlesignin';
```

Comment:
We need to create a setter method in the OAuth2 class so that here we can optionally call it. If we don't call it here, as per my comment above, the getAuthenticationSchemeAlias() method will just set it to the same this as $this->providerKey. We could call it here like this:

```
$this->setAuthenticationSchemeAlias('googlesignin');
```
but I think we should just remove this whole change to GoogleSignIn.